ceno_zkvm/benches/fibonacci.rs

```diff
@@ -43,6 +44,36 @@ fn setup() -> (Program, Platform) {
 fn fibonacci_prove(c: &mut Criterion) {
     let (program, platform) = setup();
     for max_steps in [1usize << 20, 1usize << 21, 1usize << 22] {
+        // estimate proof size data first
```

Comment:
Sorry, I don't understand.  Why do we need this estimate first?

Reply:
Yeah, I think we should move this to `e2e.rs` after we generated the proof. The benchmark we have below is only used to see if there is progress or regression. The proof size should be identical in multiple proving instances.